File: repos_estudiantes_ALVAREZ_primera_entrega/ARUQUIPA_VARGAS_ANA_GLORIA/POLIMORFISMO/Ejercicio15/Ordenador.py

class Ordenador:
    def __init__(self, codigo_serial, numero, ram, procesador, estado):
        self.codigo_serial = codigo_serial
        self.numero = numero
        self.ram = ram  
        self.procesador = procesador
        self.estado = estado 

    def mostrar(self):
        print(f"Código: {self.codigo_serial}, Nº: {self.numero}, RAM: {self.ram}GB, CPU: {self.procesador}, Estado: {self.estado}")

File: repos_estudiantes_ALVAREZ_primera_entrega/CHAMBI_BARRETO_CARLOS_ERNESTO/POLIMORFISMO/EJERCICIO15/Ordenador.java
package POLIMORFISMO.EJERCICIO15;

public class Ordenador{
    public String codigoSerial, procesador;
    public int numero, ram;
    public boolean estado;

    public Ordenador(String codigoSerialserial, String procesador, int numero, int ram, boolean estado) {
        this.codigoSerial = codigoSerialserial;
        this.procesador = procesador;
        this.numero = numero;
        this.ram = ram;
        this.estado = estado;
    }

    public boolean isEstado(){
        return estado;
    }
    public int getRam(){
        return ram;
    }
    public String getCodigoSerial(){
        return codigoSerial;
    }
    public void setActivo(boolean activo) {
        this.estado = activo;
    }

    public void mostrar() {
        System.out.println("Serial: " + codigoSerial +
                ", Número: " + numero +
                ", RAM: " + ram + "GB" +
                ", Procesador: " + procesador +
                ", Estado: " + (estado ? "Activo" : "Inactivo"));
    }
}
